src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/config/InboundEventIntegrationConfig.java

```diff
@@ -0,0 +1,187 @@
+import uk.gov.ons.ctp.common.jackson.CustomObjectMapper;
+import uk.gov.ons.ctp.common.retry.CTPRetryPolicy;
+import uk.gov.ons.ctp.integration.contactcentresvc.config.MessagingConfig.ContainerConfig;
+
+/** Integration configuration for inbound events. */
+@Configuration
+public class InboundEventIntegrationConfig {
+
+  private AppConfig appConfig;
+
+  /**
+   * Constructor for InboundEventIntegrationConfig
+   *
+   * @param appConfig centralised configuration
+   */
+  public InboundEventIntegrationConfig(final AppConfig appConfig) {
+    this.appConfig = appConfig;
+  }
+
+  /**
+   * Configure a backoff for what happens when rabbitMQ goes down. This allows us to do less
+   * thrashing than the default when there is a rabbit problem thus reducing load on the system.
+   *
+   * @return backoff object
+   */
+  @Bean
+  public BackOff rabbitDownBackOff() {
+    MessagingConfig messaging = appConfig.getMessaging();
+    BackoffConfig recoveryBackoff = messaging.getRecoveryBackoff();
+    ExponentialBackOff backoff = new ExponentialBackOff();
+    backoff.setMaxInterval(recoveryBackoff.getMax());
+    backoff.setMultiplier(recoveryBackoff.getMultiplier());
+    backoff.setInitialInterval(recoveryBackoff.getInitial());
+    return backoff;
+  }
+
+  /**
+   * Configure the retry behaviour for when a Case message throws a RuntimeException while it is
+   * being processed in our event processing code.
+   *
+   * <p>Note that this does not apply to when Rabbit goes down, just problems with the subsequent
+   * processing throwing an unchecked exception (of the type that Spring might emit).
+   *
+   * <p>Note also that the retry policy is configured with "maximum attempts" so for instance if
+   * "conMaxAttempts" is 3 , then there will be at most 2 retries after the initial attempt.
+   *
+   * @return retry template
+   */
+  @Bean
+  public RetryTemplate caseRetryTemplate() {
+    ContainerConfig messaging = appConfig.getMessaging().getCaseListener();
+    BackoffConfig processingBackoff = messaging.getProcessingBackoff();
+    ExponentialBackOffPolicy backoffPolicy = new ExponentialBackOffPolicy();
+    backoffPolicy.setMaxInterval(processingBackoff.getMax());
+    backoffPolicy.setMultiplier(processingBackoff.getMultiplier());
+    backoffPolicy.setInitialInterval(processingBackoff.getInitial());
+    RetryTemplate template = new RetryTemplate();
+    template.setBackOffPolicy(backoffPolicy);
+    RetryPolicy retryPolicy = new CTPRetryPolicy(messaging.getConMaxAttempts());
+    template.setRetryPolicy(retryPolicy);
+    return template;
+  }
+
+  /**
+   * Create advice bean for the Case listener, containing the retry configuration, and specify that
+   * if the retry fails, to place the message in the DLQ.
+   *
+   * @param caseRetryTemplate caseRetryTemplate
+   * @return retry advice for the listener.
+   */
+  @Bean
+  public StatelessRetryOperationsInterceptorFactoryBean caseRetryAdvice(
```

Comment:
Minor, but can't this be private? 
The private method would need to be called when the bean was needed of course, but it could be argued that would make construction of the SimpleMessageListenerContainer more obvious.
Comment probably applies to similar methods.

Reply:
This made a bit more sense when there was both UAC and Case Events. 
As before , I think we can look at this again if this code survives.